workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/ddm-templates/learn-search-category-facet-tabs/ddm-template.ftl

```diff
@@ -136,4 +136,105 @@
 			}
 		}
 	}
-</@>
\ No newline at end of file
+</@>
+
+<style>
+	@media screen and (max-width: 992px) {
+		.learn-category-facet-tabs .facet-value-mobile {
+			gap: var(--spacer-2, 0.5rem);
+		}
+
+		.learn-category-facet-tabs .facet-value-mobile .term-text {
+			opacity: 0.80;
+		}
+
+		.learn-category-facet-tabs .dropdown-menu,
+		.learn-category-facet-tabs#tab-list-mobile {
+			max-width: none;
+			padding: var(--spacer-2, 0.5rem);
+			width: 100%;
+		}
+
+		.learn-category-facet-tabs#tab-list {
+			display: none !important;
+		}
+
+		.learn-category-facet-tabs#tab-list-mobile {
+			align-items: center;
+			display: flex !important;
+			width: 100%;
+		}
+
+	}
+
+	.learn-category-facet-tabs .facet-term-unselected .term-text {
+		opacity: 0.8;
+	}
+
+	.learn-category-facet-tabs .facet-value {
+		flex:1;
+	}
+
+	.learn-category-facet-tabs.tab-list {
+		align-items:center;
+		display: flex;
+		background: var(--Neutral-01, #F7F7F8);
+		border-radius: 99px;
+		height: 52px;
+		padding: 4px 6px;
+	}
+
+	.learn-category-facet-tabs .selected-tab-btn {
```

Comment:
Instead of repeating classes, we could nest them. What do you think ?

Istead of  

```
.learn-category-facet-tabs .selected-tab-btn {}
.learn-category-facet-tabs .term-count {}
.learn-category-facet-tabs .term-text
```
try 
```
.learn-category-facet-tabs { 
      .selected-tab-btn {
      } 
      .term-text {
      } 
      .term-count {
      } 
}
```